### PR TITLE
ENG-6772 Add JDBC + Postgres and Vertica export system tests with data checking.

### PR DESCRIPTION
**Pull request by pdshaw**

### CODE REVIEW

File: tests/test_apps/genqa/run.sh [hunk 1]
```diff
@@ -36,8 +37,7 @@ CLASSPATH=$({ \
 # ZK Jars needed to compile kafka verifier. Apprunner uses a nfs shared path.
 ZKCP=${ZKLIB:-"/home/opt/kafka/libs"}
 RBMQ=${RBMQLIB:-"/home/opt/rabbitmq"}
-CLASSPATH="$CLASSPATH:$ZKCP/zkclient-0.3.jar:$ZKCP/zookeeper-3.3.4.jar:$RBMQ/rabbitmq.jar"
-VOLTDB="$VOLTDB_BIN/voltdb"
+CLASSPATH="$CLASSPATH:$ZKCP/zkclient-0.3.jar:$ZKCP/zookeeper-3.3.4.jar:$RBMQ/rabbitmq.jar:vertica-jdbc.jar"
 VOLTDB="$VOLTDB_BIN/voltdb"
```
Comment (prosegay) [inline tests/test_apps/genqa/run.sh]: maybe need path on vertica-jdbc.jar

File: tests/test_apps/voltkvqa/run.sh [hunk 2]
```diff
@@ -1,278 +0,0 @@
-#!/usr/bin/env bash
```
Comment (prosegay) [inline tests/test_apps/voltkvqa/run.sh]: why are we deleting this file?

File: tests/test_apps/genqa/deployment_vertica.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0"?>
+<deployment>
+    <cluster hostcount="1" sitesperhost="4" kfactor="0"/>
+    <httpd enabled="true">
+        <jsonapi enabled="true" />
+    </httpd>
+    <export>
+        <configuration enabled="true" stream="abc" type="jdbc">
```
Comment (akhanzode) [inline tests/test_apps/genqa/deployment_vertica.xml]: use better name than abc

File: tests/test_apps/genqa/run.sh [hunk 4]
```diff
@@ -36,8 +37,7 @@ CLASSPATH=$({ \
 # ZK Jars needed to compile kafka verifier. Apprunner uses a nfs shared path.
 ZKCP=${ZKLIB:-"/home/opt/kafka/libs"}
 RBMQ=${RBMQLIB:-"/home/opt/rabbitmq"}
-CLASSPATH="$CLASSPATH:$ZKCP/zkclient-0.3.jar:$ZKCP/zookeeper-3.3.4.jar:$RBMQ/rabbitmq.jar"
-VOLTDB="$VOLTDB_BIN/voltdb"
+CLASSPATH="$CLASSPATH:$ZKCP/zkclient-0.3.jar:$ZKCP/zookeeper-3.3.4.jar:$RBMQ/rabbitmq.jar:vertica-jdbc.jar"
```
Comment (akhanzode) [inline tests/test_apps/genqa/run.sh]: Dont you need a prefix env for vertica jar?

File: tests/test_apps/genqa/run.sh [hunk 5]
```diff
@@ -48,12 +48,13 @@ CLIENTLOG="clientlog"
 
 # remove build artifacts
 function clean() {
-    rm -rf obj debugoutput $APPNAME.jar $APPNAME2.jar $APPNAME3.jar $APPNAME4.jar voltdbroot voltdbroot
+    rm -rf obj debugoutput $APPNAME.jar $APPNAME2.jar $APPNAME3.jar $APPNAME4.jar $APPNAME_EXPORT.jar voltdbroot voltdbroot
```
Comment (akhanzode) [inline tests/test_apps/genqa/run.sh]: why remove voltdbroot twice?

File: tests/test_apps/genqa/run.sh [hunk 6]
```diff
@@ -112,12 +131,36 @@ function server-mysql() {
     $VOLTDB create -d deployment_mysql.xml -l $LICENSE -H $HOST $APPNAME.jar
 }
 
+
+# run the voltdb server locally with vertica connector
+function server-vertica() {
+    # if a catalog doesn't exist, build one
+    if [ ! -f $APPNAME.jar ]; then catalog; fi
+    # run the server
+    $VOLTDB create -d deployment_vertica.xml -l $LICENSE -H $HOST $APPNAME.jar
```
Comment (akhanzode) [inline tests/test_apps/genqa/run.sh]: Do you need --force here?

File: tests/test_apps/genqa/run.sh [hunk 7]
```diff
@@ -190,21 +233,25 @@ function async-benchmark() {
         --latencytarget=10
 }
 
+function clean-vertica() {
+    echo "drop table export_partitioned_table" | ssh volt15d /opt/vertica/bin/vsql -U dbadmin test1
```
Comment (akhanzode) [inline tests/test_apps/genqa/run.sh]: make a var for volt15d so moving vertica becomes easy.

File: tests/test_apps/genqa/run.sh [hunk 8]
```diff
@@ -264,6 +351,47 @@ function export-rabbitmq-verify() {
...
+    sudo chown postgres $PGTMPDIR
+    echo "PG_PATH:" $PG_PATH
+
+    alias python=python2.6
+    # Use the same version of the JDBC jar, on all platforms
+    echo "CLASSPATH:" $CLASSPATH
+
+    # Start the PostgreSQL server, in the new temp directory
+    # Note: '-o --lc-collate=C' causes varchar sorting to match VoltDB's
+    sudo su - postgres -c "$PG_PATH/initdb -A trust -D $PGTMPDIR/data --lc-collate=C"
```
Comment (akhanzode) [inline tests/test_apps/genqa/run.sh]: Do u need sudo???

**Pull request closed without merging**